Tests/SentryTests/Integrations/ANR/SentryANRTrackingIntegrationTests.swift

```diff
@@ -108,7 +108,23 @@ class SentryANRTrackingIntegrationTests: SentrySDKIntegrationTestsBase {
         
         assertNoEventCaptured()
     }
-    
+#if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
+    func testANRDetected_ButBackground_EventNotCaptured() {
+
+        class BackgroundSentryUIApplication: SentryUIApplication {
+            override var applicationState: UIApplication.State { .background }
+        }
```

Comment:
Excellent defining the class directly in here. This improves the test readability a lot 🥇